### ISSUE TITLE
Mapbox doesnt' work in Sketch 52.3

### ISSUE THREAD

**Issue by malykh** [eddiesigner/sketch-map-generator#39]
Hi there. I keep getting a message "There was a problem, please check the  address settings." But Google maps works fine.

**Comment by BossOz** [comment]
the same for me

**Comment by eddiesigner** [comment]
I've solved the issue, please update the plugin to the latest version (2.0.6)

**Comment by dawnguyen** [comment]
Google Map is fine, But Mapbox still shows this error, I updated the plugin to 2.0.6

**Comment by eddiesigner** [comment]
It works for me, what address are you trying to put?

**Comment by eddiesigner** [comment]
I will close this issue since there has not been any recent activity, please feel free to reopen it if you consider it necessary.

**Comment by akashsheshadri** [comment]
Hello, Ive faced the same issue, using the plugin 3.0.0 on sketch ver52.4. Google maps is fine, but would like to use the light and dark styles of Mapbox, and it says "there was a problem please check the address settings." Thanks!

**Comment by eddiesigner** [comment]
@akashsheshadri Please try with other address or tell me what address are you trying to use?

**Comment by akashsheshadri** [comment]
Hey, I tried inputting rosenheimerplatz Munich (doesn't preview), and just plain Munich (shows preview but does not generate the map). Also tried it with plain Berlin, didn't show. Could you show a sample address that might work in the case? thanks

**Comment by eddiesigner** [comment]
@akashsheshadri I think the correct way to make it to work is writing **rosenheimer platz**, that's how that place is named in Google Maps and Mapbox. I put that place and it worked just fine.

Also I tried with Munich and Berlin and both work. I've tested with _Peuerbachstrasse Linz_, _Linz Asutria_, _Munich_, _Salzburg_, _Gutemberg 189, Anzures, 11590 Ciudad de México, CDMX, Mexico_ and others and all of them work.
